tuned/plugins/plugin_video.py

```diff
@@ -61,6 +65,12 @@ def _set_radeon_powersave(self, value, device, sim):
 			if not sim:
 				self._cmd.write_to_file(sys_files["method"], "dynpm")
 			return "dynpm"
+		elif value == "dpm":
+			if not sim and self._get_radeon_powersave(device) != "dpm":
+				log.error("Cannot set dpm power method through the video plugin." \
+						+ "Use the sysctl plugin with the 'radeon.dpm=1' option")
```

Comment:
I don't think this is strictly correct. The radeon.dpm option is a boot-time parameter, not one that can be changed with sysctl.